include/Metabox/PostBase.php

```diff
@@ -10,5 +10,62 @@
  * @since 2.0.0
  */
 abstract class BM_Metabox_PostBase extends BM_Metabox_Base {
+	/**
+	 * Renders the categories select.
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param string $name Select tag's name.
+	 */
+	protected function render_categories_select( $name = '' ) {
```

Comment:
Name should not be an optional parameter. Also let's call this function as `render_categories_dropdown`.